build/ProjectReunion-release.yml

```diff
@@ -149,14 +142,57 @@ jobs:
       ArtifactServices_Symbol_AccountName: microsoftpublicsymbols
       ArtifactServices_Symbol_UseAAD: true
 
-  - task: PkgESCodeSign@10
-    displayName: CodeSign
-    env:
-      SYSTEM_ACCESSTOKEN: $(system.accesstoken)
+  - task: EsrpCodeSigning@1
     inputs:
-      signConfigXml: '$(Build.SourcesDirectory)\build\SignConfig.xml'
-      inPathRoot: '$(Build.ArtifactStagingDirectory)\fullnuget'
-      outPathRoot: '$(Build.ArtifactStagingDirectory)\fullnuget'
+      ConnectedServiceName: 'Project Reunion ESRP Code Signing Connection'
+      FolderPath: '$(Build.ArtifactStagingDirectory)\fullnuget'
+      Pattern: |
+        lib\uap10.0\Microsoft.ApplicationModel.Resources.winmd
```

Comment:
(non-blocking for now), can we just say *.dll and *.winmd or does this not support wildcards?  If it doesn't support wildcards I'm wondering if we can take the artifact drop output and write a task that spits the dll and winmd names so we don't have to hard-code this.

Reply:
It does support wild cards but since we were specifying it in build/signconfig.xml, I thought I should specify it here too.

Reply:
Another thing is that since the new ESRP task does not support using sign config xml files, there will ne no need for build/SignConfig.xml and build/NugetSignConfig.xml. I will be removing it soon in the next commit.